qcut/.agents/skills/libtv-skill/scripts/download_results.py

```
@@ -0,0 +1,154 @@
+    seen = set()
+    unique = []
+    for u in urls:
+        if u not in seen:
+            seen.add(u)
+            unique.append(u)
+    return unique
+
+
+def download_file(url, filepath):
+    """下载单个文件"""
+    req = urllib.request.Request(url, headers={"User-Agent": "LibTV-Skill/1.0"})
```

Ruff (0.15.13)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)